### DISCUSSION TITLE
Alias for Alexa

### DISCUSSION THREAD

**Discussion by jove01** [t0bst4r/home-assistant-matter-hub#306] [Q&A] [answered]
Is it possible to define a alias, that I can use in Alexa

**Comment by t0bst4r** [comment, marked as answer]
You want to rename a single device?
You can directly rename it in Alexa, if needed.

**Reply by jove01** [reply]
> You want to rename a single device? You can directly rename it in Alexa, if needed.

Thank you, this way i have done, but my hope was, there is a simalar way Like the voice Assistent

**Reply by hpuac** [reply]
> You want to rename a single device? You can directly rename it in Alexa, if needed.

In the Amazon Alexa Home Assistant integration you can define an alternative name in the `entity_config`. 
See: https://www.home-assistant.io/integrations/alexa.smart_home/#entity_config
It is much more convenient to define an alternative name in HA instead of the Alexa App.
As many use this Add-On as a drop in replacement it would be awesome if it would support the same.
But I guess there are other priorities right now. But maybe it's something for a feature list for the future 🙂 

Awesome work btw! 🙂

**Reply by jove01** [reply]
Till now I use this Integration over AWS. But I didn't know the function for renaming there.
In Matter, I think, IT IS a little bit more simple to add new devices/ entities. And there is No way over AWS.
In next time i must decide, what ist the best way for me to use Alexa.

**Reply by t0bst4r** [reply]
Someone also created an issue for that. Let's then track it there. It's low priority, but anyway.
